.github/workflows/pull_request.yml

```
@@ -23,6 +23,12 @@ jobs:
       linux_6_1_arguments_override: "--explicit-target-dependency-import-check error -Xswiftc -require-explicit-sendable"
       linux_nightly_6_0_arguments_override: "--explicit-target-dependency-import-check error -Xswiftc -require-explicit-sendable"
       linux_nightly_main_arguments_override: "--explicit-target-dependency-import-check error -Xswiftc -require-explicit-sendable"
+  
+  wasm-sdk:
+    name: WebAssembly SDK
+    # TODO: Switch to this line after https://github.com/apple/swift-nio/pull/3159/ is merged
+    # uses: apple/swift-nio/.github/workflows/wasm_sdk.yml@main
+    uses: kateinoigakukun/swift-nio/.github/workflows/wasm_sdk.yml@katei/add-wasm-ci
```

Before merging, this will need changed as follows. Leaving as is until dependencies and testing are in order.

```suggestion
    uses: apple/swift-nio/.github/workflows/wasm_sdk.yml@main
```

---

https://github.com/apple/swift-nio/pull/3331
https://github.com/apple/swift-nio/pull/3332